test/unit/bind-elements.html

```diff
@@ -486,3 +486,35 @@
     });
   </script>
 </dom-module>
+
+<dom-module id="x-bind-computed-property">
+  <template>
+    <div id="check">[[translate('Hello World.')]]</div>
```

Comment:
Could you also add a test that shows `translate` called with a non-static argument, e.g. `[[translate(foo)]]` where `foo` has a default value, but `translate` is computed async?  This is to cover the case that a dependency to computing the `translate` function is async (e.g. due to loading a string bundle via an xhr).  Although all args are valued, the method is still undefined, and it should not warn.

I believe your `bailoutEarly` flag covers this case, but would be good to test.